build.xml

```diff
@@ -80,6 +81,11 @@
     <pathelement location="lib/profiler-parser-shaded-1.4.1.jar" />
     <pathelement location="lib/ast.jar" />
     <pathelement location="lib/ast-dependencies.jar" />
+
+    <!-- ABLDuck classes -->
+    <pathelement location="lib/antlr-4.7-complete.jar"/>
+    <pathelement location="lib/markdown4j-2.2.jar"/>
+  	<pathelement location="${build-javadoc}" />
```

Comment:
Haha ah! Thats where i've been going wrong :rofl: sorry i normally have my editor set to replace tabs with spaces but i see i didn't do this after reinstalling.